keyservice/keyservice.proto

```diff
@@ -17,12 +18,19 @@ message KmsKey {
 	string arn = 1;
 	string role = 2;
 	map<string, string> context = 3;
+	string aws_profile = 4;
```

Comment:
This seems unrelated to this PR?

Reply:
Yeah, but this is necessary. Without this regenerating go sources from this `proto` ends up removing `AwsProfile` from the code, which breaks the build!

Perhaps someone has directly modified the source without adding this to proto.

Reply:
Interesting. I guess they forgot to commit the .proto file. Thanks for clarifying!